enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/server/HttpRequestHandler.java

```diff
@@ -74,6 +76,14 @@ private RequestContext buildRequestContext(API api, CheckRequest request) {
                 request.getAttributes().getSource().getAddress().hasSocketAddress()) {
             address = request.getAttributes().getSource().getAddress().getSocketAddress().getAddress();
         }
+        if (!request.getAttributes().getRequest().getHttp().getRawBody().isEmpty()) {
+            ByteString byteString = request.getAttributes().getRequest().getHttp().getRawBody();
+            requestBody = byteString.toStringUtf8();
```

Comment:
Do we need to call isValidUtf8 prior to assigning it as utf8 string? https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/ByteString.html#toStringUtf8--